### PR TITLE
qemu_guest_agent:clear readonly bit for windows disk and online them

### PR DESCRIPTION
**Pull request by xiagao**
Clear readonly for disks and online them in windows guest before format them.

ID: 1658948
Signed-off-by: Xiaoling Gao <xiagao@redhat.com>

### CODE REVIEW

**Comment by XueqiangWei** [comment]
LGTM,
Acked-by: xuwei@redhat.com

**Comment by xiagao** [comment]
'1-Host_RHEL.mx.u0.qcow2.virtio_scsi.up.virtio_net.Guest.Win10.x86_64.io-github-autotest-qemu.qemu_guest_agent.virtio_serial.check_hotplug_frozen.q35' Finshed 2018-12-19 11:18:04 2018-12-19 11:25:35 PASS 150.472899

'1-Host_RHEL.mx.u0.qcow2.virtio_scsi.up.virtio_net.Guest.Win2016.x86_64.io-github-autotest-qemu.qemu_guest_agent.virtio_serial.check_hotplug_frozen.q35' Finshed 2018-12-19 16:54:14 2018-12-19 16:56:14 PASS 120.047167

**Comment by kimi1978wy** [comment]
Acked-by: <wyu@redhat.com>

File: qemu/tests/qemu_guest_agent.py [hunk 1]
```diff
@@ -1163,6 +1163,9 @@ def get_new_disk(disks_before_plug, disks_after_plug):
                 disk_index = utils_misc.wait_for(
                     lambda: utils_disk.get_windows_disks_index(session, image_size_stg0), 120)
                 if disk_index:
+                    logging.info("Clear readonly for disk and online it in windows guest.")
+                    if not utils_disk.update_windows_disk_attributes(session, disk_index):
+                        test.error("Failed to update windows disk attributes.")
```
Comment (YongxueHong) [inline qemu/tests/qemu_guest_agent.py]: For ling 1168("test.error("Failed to update windows disk attributes.")"), It is one step of this case ?  suggest to call the method test.fail if it is a step indeed.
Thanks.

Comment (xiagao) [reply qemu/tests/qemu_guest_agent.py]: No,it's not a step.

**Comment by YongxueHong** [comment]
Thanks all.

**Pull request merged**